.github/workflows/benchmark.yml

```diff
@@ -22,15 +22,11 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Setup uv
```

Comment:
@mwtoews the way I read the uv docs, the reason to keep `setup-python` after `setup-uv` is the cache restore might be faster. No strong feelings about this though.

Reply:
Yes, the docs say that, but I don't see any time difference. I believe it picks the Python version off the runner [image](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#python), which should be just as fast (provided it is included with the image). Maybe the setup-python action is preferred if other versions of Python are required.